ghost/core/test/unit/server/models/post.test.js

```diff
@@ -887,60 +887,6 @@ describe('Unit: models/post: uses database (@TODO: fix me)', function () {
                     });
                 });
 
-                it('rejects if changing authors', function (done) {
```

Comment:
[To help reviewers] The change in [this PR](https://github.com/TryGhost/Ghost/pull/14586/files#diff-c351cb589adefbb886570cfadb33b33eb8fdc12bde1024d1188cd18c165fc5e8L1010) made three tests here mostly the same. Deduplicated them and kept only one.